src/module-1/cutStrings/index.js

```diff
@@ -1 +1,16 @@
-export const cutStrings = (arr = []) => {};
+export const cutStrings = (arr = []) => {
+	let min = Infinity;
```

Comment:
You need to find the lowest value, why don't you just set `arr[0].length` as the smallest one and then check if other values are less than the min. 
No need to use `Infinity`, just compare values between each other.

To find min value you could also use `reduce` operator.

Reply:
I used `Math.min`.